ganttproject/src/net/sourceforge/ganttproject/gui/GanttDialogPerson.java

```diff
@@ -78,6 +81,7 @@ public GanttDialogPerson(CustomPropertyManager customPropertyManager, TaskManage
 
     ((GPAbstractOption)myTotalCostField).setWritable(false);
     myRateGroup = new GPOptionGroup("resourceRate", new GPOption[] {myStandardRateField, myTotalCostField});
+    myRate2Group = new GPOptionGroup("resourceRate", new GPOption[] {myStandardRate2Field, myTotalCost2Field});
```

Comment:
The same normally applies to option groups: use different ids for different groups